pom.xml

```diff
@@ -82,6 +83,13 @@
         <artifactId>ingestion-client</artifactId>
         <version>${streamx.version}</version>
       </dependency>
+      <!-- This version needs to be same as in ingestion-client -->
```

Comment:
Can we upgrade the version in ingestion-client instead?
We should rather make our code compatible with new releases, as we won't be able to stick with 1.11.3.